checkstyle.properties

```diff
@@ -1,14 +1,20 @@
 category=External Analysers
 description=Provide Checkstyle rules for Java projects
 homepageUrl=https://github.com/checkstyle/sonar-checkstyle
-archivedVersions=10.5,10.6.0,10.7.0,10.8.1,10.9.3,10.11.0,10.12.0,10.12.1,10.12.3,10.12.5,10.14.1
-publicVersions=10.14.2
+archivedVersions=10.5,10.6.0,10.7.0,10.8.1,10.9.3,10.11.0,10.12.0,10.12.1,10.12.3,10.12.5,10.14.1,10.14.2
+publicVersions=10.15.0
 
 defaults.mavenGroupId=org.sonarsource.checkstyle
 defaults.mavenArtifactId=sonar-checkstyle-plugin
 
+10.15.0.description=Upgrade to Checkstyle 10.15.0
+10.15.0.sqVersions=[9.9,LATEST]
+10.15.0.date=2024-04-11
+10.15.0.changelogUrl=https://github.com/checkstyle/sonar-checkstyle/releases/tag/10.15.0
+10.15.0.downloadUrl=https://github.com/checkstyle/sonar-checkstyle/releases/download/10.15.0/checkstyle-sonar-plugin-10.15.0.jar
+
 10.14.2.description=Upgrade to Checkstyle 10.14.2
-10.14.2.sqVersions=[9.9,LATEST]
+10.14.2.sqVersions=[9.9,10.4]
```

Comment:
Would you end this range with either `10.4.*` or `10.4.1` so your users on 10.4.1 will be prompted to upgrade?

Reply:
i changed it to `10.4.*`. 😃